### PR TITLE
Remove MIOpen integration from PR CI

### PR DESCRIPTION
**Pull request by yiqian1**

### CODE REVIEW

File: mlir/utils/jenkins/Jenkinsfile [hunk 1]
```diff
@@ -465,21 +472,40 @@ pipeline {
...
+                        }
+                    }
+                    stage("Static Library: build rockMLIR packages") {
                         when {
                             beforeAgent true;
                             equals expected: true, actual: params.staticLib;
                             equals expected: false, actual: params.nightly;
                         }
-                        environment {
+                        //environment {
```
Comment (krzysz00) [inline mlir/utils/jenkins/Jenkinsfile]: Why commented out?

File: mlir/utils/jenkins/Jenkinsfile [hunk 2]
```diff
@@ -465,21 +472,40 @@ pipeline {
...
-                            LD_LIBRARY_PATH="${WORKSPACE}/MIOpen/build/lib:$LD_LIBRARY_PATH:"
-                        }
+                        //    LD_LIBRARY_PATH="${WORKSPACE}/MIOpen/build/lib:$LD_LIBRARY_PATH:"
+                        //}
                         steps {
                             sh 'rm build/CMakeCache.txt'
-                            buildMIOpenWithMLIR()
-                            testMIOpenDriver(/*testInt8=*/true, /*tuning=*/false, "selected-resnet50-miopen-configs")
-                            testMIOpenDriver(/*testInt8=*/true, /*tuning=*/true, "selected-resnet50-miopen-configs")
+                            buildProject('package', '-DBUILD_FAT_LIBROCKCOMPILER=ON')
```
Comment (krzysz00) [inline mlir/utils/jenkins/Jenkinsfile:499]: Might be woth making this MIXR_TARGET or whatever flag it is they use over there?

**Review by krzysz00 — COMMENTED** [review]
Nitpicks, plan is solid

File: mlir/utils/jenkins/Jenkinsfile (hunk 2 above)
Comment (yiqian1) [reply mlir/utils/jenkins/Jenkinsfile:499]: Yes, and need to ensure a new target would still use the default devel packages.

File: mlir/utils/jenkins/Jenkinsfile [hunk 3]
```diff
@@ -153,6 +153,13 @@ void testMIOpenDriver(boolean testInt8, boolean tuning, String configFilename) {
     }
 }
 
+//viod buildAndTestMIopenDriver() {
```
Comment (krzysz00) [inline mlir/utils/jenkins/Jenkinsfile]: Don't comment things out, just remove them.

File: mlir/utils/jenkins/Jenkinsfile [hunk 4]
```diff
@@ -465,21 +472,36 @@ pipeline {
...
                             beforeAgent true;
-                            equals expected: true, actual: params.staticLib;
+                            equals expected: true, actual: params.sharedLib;
                             equals expected: false, actual: params.nightly;
                         }
-                        environment {
-                            // Make libMIOpen.so accessible to the test driver
-                            LD_LIBRARY_PATH="${WORKSPACE}/MIOpen/build/lib:$LD_LIBRARY_PATH:"
+                        steps {
+                            buildProject('ci-performance-scripts', '')
```
Comment (krzysz00) [inline mlir/utils/jenkins/Jenkinsfile:475]: Let's get a timeout around this?

File: mlir/utils/jenkins/Jenkinsfile.downstream [hunk 5]
```diff
@@ -106,6 +106,22 @@ pipeline {
                                 """)
                              }
                         }
+                        stage("Tune selected rocmlir configs") {
+                            steps {
+                                buildProject('ci-performance-scripts', '')
+                                dir('build') {
```
Comment (krzysz00) [inline mlir/utils/jenkins/Jenkinsfile.downstream:112]: Same note re timeout

File: mlir/utils/jenkins/ci-configs/selected-conv-configs [hunk 6]
```diff
@@ -0,0 +1,2 @@
+-n 64 -c 1024 -H 14 -W 14 -k 256 -y 1 -x 1 -p 0 -q 0 -u 1 -v 1 -l 1 -j 1 -m conv -g 1 -t 1
+#-n 256 -c 256 -H 14 -W 14 -k 256 -y 3 -x 3 -p 1 -q 1 -u 1 -v 1 -l 1 -j 1 -m conv -g 1 -t 1
```
Comment (krzysz00) [inline mlir/utils/jenkins/ci-configs/selected-conv-configs]: Why's this commented out?

**Review by krzysz00 — COMMENTED** [review]
I've got some more questions/notes.

File: mlir/utils/jenkins/ci-configs/selected-conv-configs (hunk 6 above)
Comment (yiqian1) [reply mlir/utils/jenkins/ci-configs/selected-conv-configs]: I just want one conv config. Will remove it.

**Review by krzysz00 — APPROVED** [review]

**Comment by krzysz00** [comment]
I'm landing this to unbreak our CI, which seems to be hitting a MIOpen build failure.

**Pull request merged**